lib/aliquot-pay.rb

```diff
@@ -181,23 +216,21 @@ def build_signature
             ensure_intermediate_key
           end
 
-    signature_string =
-      signed_string_message = ['Google',
-                               recipient_id,
-                               @protocol_version.to_s,
-                               signed_message_string].map do |str|
-        [str.length].pack('V') + str
-      end.join
+    signature_string = ['Google',
+                        recipient_id,
+                        @protocol_version.to_s,
+                        signed_message_string].map do |str|
+      [str.length].pack('V') + str
+    end.join
     @signature = sign(key, signature_string)
   end
 
   def build_signatures
     return @signatures if @signatures
 
-    signature_string =
-      signed_key_signature = ['Google', 'ECv2', signed_key_string].map do |str|
-        [str.to_s.length].pack('V') + str.to_s
-      end.join
+    signature_string = ['Google', 'ECv2', signed_key_string].map do |str|
+      [str.to_s.length].pack('V') + str.to_s
```

Comment:
Is `#to_s` really necessary? Isn't signed_key_string always a string? You only use it for `@protocol_version` in the similar code above, not for `signed_message_string`.

Reply:
Similar to https://github.com/clearhaus/aliquot-pay/pull/16/files/bcb2d87a66aa95930f5a94f6e3cd0d629da1dd8c#r1483007351 i only remove what I thought was a superfluous assignment.

Reply:
Completely fair :+1: